src/commands/create.ts

```diff
@@ -51,13 +64,29 @@ export default class Create extends Command {
 
         this.log('');
 
-        const folder = path.join(process.cwd(), info.nameSlug);
+        let folder = path.join(process.cwd(), info.nameSlug);
+        const fd = new FolderDetails(this);
+        const { flags, args }  = this.parse(Create);
+        const sourcePath = path.resolve(args.source);
+        if (await fd.doesFolderExist(sourcePath)) {
+            folder = path.join(sourcePath, info.nameSlug);
+        } else {
+            this.log(chalk.bgRed(`\nInvalid source path provided. Could not resolve: "${ args.source }"`));
```

Comment:
If a relative path is provided, we will create a new folder for the user, but warn him if the folder already exists and may overwrite the content unexpectedly. What do you think of it?

Reply:
![image](https://user-images.githubusercontent.com/32427260/75788766-a5be8380-5da3-11ea-9fcb-02b3e561d87d.png)
FYI

Reply:
@lolimay the path provided is basically the folder in which the app-folder is created, it isn't overwritten but the new-apps-folder is nested within it. So if the folder doesn't exist I am not creating the folder, but rather reporting that the folder doesn't exist.

Reply:
So basically what  I have done is allow the user to create the apps-folder in any folder in which he wants and which exist but the folder's name remains the default, but what git has is that it takes the relative path and as well as the name of the folder to be created.
So what do you think, is it better to switch to the git approach or this?

Reply:
Would like to see your thoughts on it @d-gubert @graywolf336

Reply:
I have to admit I thought it would behave like `git` when you passed that argument 😬 I think the way it is implemented now is a bit confusing/misleading.

Maybe if you change the `source` argument to be a `destination`  instead (like `git`) it would be much more intuitive to use

Reply:
Ok, changed the variable name and changed source to destination

Reply:
Is this fine?

Reply:
I think Douglas's idea is that it's better if we can switch to the git approach. @ritwizsinha can you please make the changes?

Reply:
Ok I'll make them

Reply:
Actually I realized that when we create an app we are already asked to name the app which is then set to the name of the folder, so we don't actually need to provide name of the folder but just the destination.

Reply:
Hence the use of the method as in git, would be redundant, what do you think @lolimay @d-gubert

Reply:
I don't think so, the name we asked will be added to the `app.json` (which be displayed in the admin - apps pane) and determine the name the entry point source file's name. It's ok if we can specify the `destination`.

Reply:
> I don't think so, the name we asked will be added to the `app.json` (which will be displayed in the admin - apps pane) and determine the name the entry point source file's name. It's ok if we can specify the `destination`.

Sorry if I am misunderstanding but apart from the entry point file, it is naming the folder also apart from the entry file. The github folder is created as the name of the app specified was github
![image](https://user-images.githubusercontent.com/43509699/75898520-2ba00480-5e60-11ea-9eac-3b1ceecb877f.png)

Reply:
exactly. but if we want the app's name `An Awesome App` but the source folder's name `awesome-app`, we can use `destination` to specify the folder's name.

Reply:
In that case if we want the app in a different folder but with the same name as appName we wouldn't be able to do so without explicitly naming it

Reply:
Thanks I get it now, what I have implemented is the ability to create the directory in a different folder but what is needed is to have the ability to change folder name

Reply:
Made all the changes